src/app/entity-metadata.ts

```diff
@@ -17,6 +17,7 @@ export const entityMetadata: EntityMetadataMap = new Map([
       icon: ObservabilityIconType.Service,
       detailPath: (id: string) => ['services', 'service', id],
       listPath: ['services'],
+      apisListPath: (id: string) => ['services', 'service', id, 'endpoints'],
```

Comment:
Hmm - rather than introducing this which is only relevant to a small subset of entities, could we concatenate the detail path of one entity with the list path of the api entity? It would make the assumption that the list path is always at the same subroute, but I think that's the behavior we want anyway.

Reply:
I did it this way so that we'd have flexibility on where the list path is. Moreover, it would keep everything in _configs_ without having to add a subroute information to the resolver code.